pkg/prometheus/host_filter_test.go

```diff
@@ -49,71 +49,71 @@ func TestFilterGroups(t *testing.T) {
 		{
 			name: "kubneretes host match",
 			group: makeGroup([]model.LabelSet{{
-				model.AddressLabel:                       "mycontainer",
```

Comment:
I was expecting to see a mixture, some tests would check against the pod label and others against the node label. I'm all in favour of not duplicating the test if they'd check against one or the other in the same way but curious on whenever this is tested somewhere else or you just decided to that one is more important the other.

Reply:
I was lazy :) Let me change this.